internal/services/mssql/helper/elasticpool.go

```diff
@@ -201,34 +201,52 @@ var getvCoreMaxGB = map[string]map[string]map[int]float64{
 	},
 	"hyperscale": {
 		"gen5": {
-			4:  1024,
-			6:  1536,
-			8:  2048,
-			10: 2048,
-			12: 3072,
-			14: 3072,
-			16: 3072,
-			18: 3072,
-			20: 3072,
-			24: 4096,
-			32: 4096,
-			40: 4096,
-			80: 4096,
+			4:  128000,
+			6:  128000,
+			8:  128000,
+			10: 128000,
+			12: 128000,
+			14: 128000,
+			16: 128000,
+			18: 128000,
+			20: 128000,
+			24: 128000,
+			32: 128000,
+			40: 128000,
+			80: 128000,
 		},
 		"prms": {
-			4:  1024,
-			6:  1536,
-			8:  2048,
-			10: 2048,
-			12: 3072,
-			14: 3072,
-			16: 3072,
-			18: 3072,
-			20: 3072,
-			24: 4096,
-			32: 4096,
-			40: 4096,
-			80: 4096,
+			4:   128000,
+			6:   128000,
+			8:   128000,
+			10:  128000,
+			12:  128000,
+			14:  128000,
+			16:  128000,
+			18:  128000,
+			20:  128000,
+			24:  128000,
+			32:  128000,
+			40:  128000,
+			64:  128000,
+			80:  128000,
+			128: 128000,
+		},
+		"moprms": {
+			4:  128000,
+			6:  128000,
+			8:  128000,
+			10: 128000,
+			12: 128000,
+			14: 128000,
+			16: 128000,
+			18: 128000,
+			20: 128000,
+			24: 128000,
+			32: 128000,
+			40: 128000,
+			64: 128000,
+			80: 128000,
```

Comment:
Same here, we should be setting meaningful values here. The Azure documentation should probably have some information on what the values of these should be.

Reply:
@stephybun They are not set to arbitrary maximum value. They are set to 128TB see https://learn.microsoft.com/en-us/azure/azure-sql/database/resource-limits-vcore-single-databases?view=azuresql#hyperscale---provisioned-compute---gen5 for more details.